src/octopeer-github/main/ElementSelectionBehaviour/NameElementSelectionBehaviour/NameElementSelectionBehaviour.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path="../ElementSelectionBehaviour.ts"/>
+/// <reference path="../../DatabaseAdaptable/DatabaseAdaptable.ts"/>
+/// <reference path="../../ElementEventBinding/ElementEventBinding.ts"/>
+/**
+ * Created by Mathias on 2016-05-11.
+ */
+class NameElementSelectionBehaviour implements ElementSelectionBehaviour {
```

Comment:
Yes. But here where some problems with the connection with the Controller. When making the classes abstract, the Controller gives errors. Therefore I left it like this, will make another issue to fix this in the future